### DISCUSSION TITLE
[Question] working with MRTK Unity?

### DISCUSSION THREAD

**Discussion by lekhachoa** [StereoKit/StereoKit#280] [Q&A]
Hello,
just a question: Can I use Stereo Kit with Unity, MRTK  ? I love the simple of Stereo Kit. But the UI/UX  of MRTK is beautiful. My current project use Unity. So, can I combine all of it?

**Comment by maluoi** [comment]
Unity and StereoKit are both immersive engines that control the system and display, unfortunately you have to choose one or the other.

I do have plans to bring StereoKit's UI a little more in line with the MRTK though, so depending on what you're looking for, you may find it in StereoKit eventually. Could you elaborate a little more on what you like about MRTK? Is it primarily the visuals, or particular UI elements?

**Reply by lekhachoa** [reply]
> Thank for your reply. 
Is it primarily the visuals. 
Besides it, could I use power of PC to use for my app? This is what so call "Holographic Remoting" from Microsoft, in which all the heavy process take places on PC, and PC will stream Holographic to Hololens

**Reply by maluoi** [reply]
Thanks for the details! And I suppose keep your eyes out for future versions of SK, I'll definitely be doing a revision on the UI visuals. As a side note here, if you're an artist or shader capable, SK's UI is pretty customizable! You could already achieve a look similar to MRTK (Or something of your own!) with a bit of extra effort. 

StereoKit doesn't currently do Holographic Remoting on its own, I've yet to really delve into the specifics of the API. I'm tracking the feature over at #225, but it's not currently towards the top of my priority list right now.

**Reply by lekhachoa** [reply]
Great !! That sound really good to me. I'll keep following the update of SK. The simplicity of your API impresses me so much.
Just a suggestion, Could you release Theme for UI visuals?. I think It's quite interessting. 
And whenever you release update with supporting Holographic Remoting, I would love to try
Thank you very much
